src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -36,8 +36,13 @@
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class Result extends DomainObject {
 
+    /*
+     * IMPORTANT on removal: Mention in release notes that an intermediary upgrade to version 5.8.5 or later is necessary before updating to the current version, as otherwise the
```

Comment:
```suggestion
     * IMPORTANT on removal: Mention in release notes that an intermediary upgrade to version 5.8.6 or later is necessary before updating to the current version, as otherwise the
```